playbook/app/controllers/playbook/pages_controller.rb

```diff
@@ -31,6 +32,15 @@ def enable_dark_mode
 
     def getting_started; end
 
+    def changelog
+      markdown = Redcarpet::Markdown.new(Redcarpet::Render::HTML, extensions = {})
+      @data = File
+              .read("../../CHANGELOG.md")
+              .to_s
+              .split(/##\s\[/)
+              .map { |change| markdown_chunk(markdown, change) }[1..-1]
+    end
```

Comment:
@bh247484 Can you weigh in here. The changelog is already in markdown. It seems strange to change this toa string then convert back to markdown.

I'm wondering if we should just display the markdown in as is in a single card.  markdown(../../CHANGELOG.md)`

Example in view: https://github.com/powerhome/playbook/blob/4f64274c26be3581583ef59840496c2e3322eba2/playbook/app/views/playbook/pages/kit_show.html.erb#L4

Reply:
Agreed.  I like the idea of splitting `CHANGELOG.md` into each constituent release but until we find other ways to get value out of that (beyond splitting them into separate cards) I think just outputting the `.md` into a single card makes more sense.

Again, cool idea though.  Would like to revisit it in the future.

Reply:
@jasoncypret

1) The conversion is a bit odd, but I'm unsure of a more safe/ straightforward way todo this. It probably is just a me being new dev issue and can be done more thoughtfully! 😅 I would love feedback on that portion in particular - it seems to also slow down the page load a tad since there is regex applied to a rather large file, though in the demo environment I didn't notice the slow-down...

2)  I'll play around with having it be displayed in one card and just displaying the markdown as-is and post it here!

Reply:
@jasoncypret @bh247484 added the screenshot for the single card view in the design feedback room [here](https://nitro.powerhrg.com/connect/api/v4/files/1615521/body). Also just merged the code here. I tried to just do `markdown("../../CHANGELOG.md") but it's expecting a file so I still am doing a `File.read` call

Lmk what ya'll think!